dcc-portal-ui/.jshintrc

```diff
@@ -15,7 +15,7 @@
   "freeze": true,
   "undef": true,
   "unused": true,
-  "strict": true,
+  "strict": false,
```

Comment:
Everything is strict now with the bundler, no need to write it ourselves, can't even disable it